src/System Application/App/Azure AD Plan/src/PlanUpgrade.Codeunit.al

```diff
@@ -207,6 +208,39 @@ codeunit 9057 "Plan Upgrade"
         UpgradeTag.SetUpgradeTag(PlanUpgradeTag.GetMicrosoft365UpgradeTag());
     end;
 
+    [NonDebuggable]
```

Comment:
Any reason this is non-debuggable?

Reply:
To be honest I can't see any reason in particular, but every other function in this codeunit that does similar upgrade is NonDebuggable